decidim-core/spec/shared/with_promoted_participatory_processes_examples.rb

```diff
@@ -53,6 +53,7 @@
 
       expect(controller.helpers.promoted_participatory_processes.count).to eq(3)
       expect(controller.helpers.promoted_participatory_processes).not_to include(unpublished)
+      expect(controller.helpers.promoted_participatory_processes).not_to include(unpromoted)
```

Comment:
I know I'm late, but I usually do `expect(array).to match_array([first, second, last])`.

The `match_array` method checks that the values of the arrays are the same, without considering the order. This way I don't need to specifically check that some values do not appear in the checked array.